ledger/controllers/wallet.js

```diff
@@ -323,6 +326,9 @@ const write = function (runtime, apiVersion) {
       }
     }
 
+    let { grants } = wallet
```

Comment:
not sure how this syntax works seeing as grants is not an attribute on wallet documents

Reply:
as discussed on slack, `grants` is also a field on the wallet documents.  when claimed, a grant is moved to the wallet document.